contributing-guides/git-terminal.md

```diff
@@ -29,3 +31,28 @@ Typically a pull request will include changes in a single file.
 [pr-howto]: ../CONTRIBUTING.md#submitting-a-pull-request
 [commit-msg]: ../CONTRIBUTING.md#commit-message
 [mass-changes]: https://github.com/tldr-pages/tldr/pulls?&q=is:pr+is:merged+label:"mass+changes"
+
+# Updating your fork
+
+Forks aren't updated automatically and in order to avoid merge conflicts it should be done manually and regularly. This is possible via the GitHub web interface by pressing `Fetch upstream` and `Fetch and merge` on the fork:
+
+![Fetch and merge button in GitHub](https://user-images.githubusercontent.com/50295997/119873818-393e2000-bf25-11eb-8a0a-b804c293439b.png).
+
+Alternatively it can also be done using Git in the terminal:
+
+```bash
+git checkout main
+git remote add upstream https://tldr-pages/tldr.git # only run this if you don't already have the upstream remote for https://github.com/tldr-pages/tldr.git
+git fetch upstream main
+git rebase upstream/main
+git push --force # not needed if you only want to update your local repository
```

Comment:
```suggestion
git checkout main
git remote add upstream https://tldr-pages/tldr.git  # only run if you don't already have the upstream remote (check with "git remote -v")
git fetch upstream main
git rebase upstream/main
git push --force-with-lease  # not needed if you only want to update your local repository
```
`--force-with-lease` is better actually, https://stackoverflow.com/a/52823955/

Reply:
Yeah, I like it.

Reply:
Why not use `` ` ` `` for `git remote -v`?

Reply:
Well it's already in a code block, so it won't look great...

Reply:
Ah, forgot about that 😄